download/index.html

```diff
@@ -0,0 +1,133 @@
+            <div class="seperator"></div>
+            <a class="section-card-link-download-page" href="https://github.com/decred/decred-binaries/releases/download/v1.0.1/decred_1.0.1-release_x86.msi" target="_blank">Win 32-bit ↓</a>
+          </div>
+        </div>
+        <div class="section-card-download-page">
+          <div class="decrediton section-card-title-download-page">Decrediton</div>
+          <div class="section-card-description-download">Graphic UI wallet for Mac OS X andLinux users.</div>
+          <div class="section-card-download-links-download-page w-clearfix">
+            <a class="section-card-link-download-page" href="https://github.com/decred/decred-binaries/releases/download/v1.0.1/decrediton-1.0.1.dmg" target="_blank">macOS ↓</a>
+            <div class="seperator"></div>
+            <a class="section-card-link-download-page" href="https://github.com/decred/decred-binaries/releases/download/v1.0.1/decrediton-1.0.1.tar.gz" target="_blank">Linux 64-bit ↓</a>
+          </div>
+        </div>
+        <div class="section-card-download-page">
+          <div class="section-card-title-download-page webwallet">Webwallet</div>
+          <div class="section-card-download-links-download-page w-clearfix">
+            <a class="section-card-link-download-page" href="https://wallet.decred.org/" target="_blank">Create a browser based Wallet →</a>
+          </div>
+          <div class="section-card-description-download">No downloads required.</div>
+        </div>
+        <div class="section-card-download-page">
+          <div class="commandline section-card-title-download-page">Command-line app suite</div>
+          <div class="section-card-description-download">A cross-platform, automatic installer/updater for the command-line applications.</div>
+          <div class="section-card-download-links-download-page w-clearfix">
+            <a class="section-card-link-download-page" href="https://github.com/decred/decred-release/releases/tag/v1.0.1" target="_blank">View on Github →</a>
+          </div>
+        </div>
+      </div>
+      <div class="section-download-page w-clearfix">
+        <div class="section-title-download-page">MINING SOFTWARE</div>
+        <div class="section-card-download-page">
+          <div class="section-card-title-download-page">gominer</div>
+          <div class="section-card-description-download">gominer is the easiest miner for Decred.</div>
+          <div class="section-card-download-links-download-page w-clearfix">
+            <a class="section-card-link-download-page" href="https://github.com/decred/gominer/releases/" target="_blank">gominer releases →</a>
+          </div>
+        </div>
+        <div class="section-card-download-page">
+          <div class="section-card-title-download-page">cgminer & ccminer</div>
+          <div class="section-card-description-download">Official builds of ccminer and cgminer are available on GitHub</div>
```

Comment:
I would just make this Unofficial - Ccminer and link to https://github.com/tpruvot/ccminer/releases like you did with sgminer.

The official builds are really old and don't work anymore anyway.

Ccminer - doesn't work anymore
cgminer - Never really worked well anyway.